src/main/java/org/apache/stanbol/commons/jsonld/JsonSerializer.java

```diff
@@ -174,7 +174,12 @@ private static void appendValueOf(Object object, StringBuffer sb, int indent, in
 	}
 
 	private static void appendList(List<Object> jsonArray, StringBuffer sb, int indent, int level) {
-		sb.append('[');
+	    sb.append('[');
+	    if(jsonArray.isEmpty()) {
```

Comment:
it is better to have clear separation for the empty arrays, please move this iff statement to the first line of code in the method

Reply:
Ok, makes sense.